samples/MyCRM.Lodgement.Core/LixiPackageSamples/NewLoanApplication.json

```diff
@@ -0,0 +1,1730 @@
+            "UniqueID": "Address_8247579",
+            "Standard": {
+              "StreetName": "Strawberry",
+              "StreetNumber": "20-22",
+              "StreetType": "Road"
+            }
+          },
+          {
+            "AustralianPostCode": "4000",
+            "AustralianState": "QLD",
+            "City": "Brisbane City",
+            "Country": "AU",
+            "Suburb": "Brisbane City",
+            "Type": "Standard",
+            "UniqueID": "Address_11087379",
+            "Standard": {
+              "StreetName": "Quay",
+              "StreetNumber": "111",
+              "StreetType": "Street"
+            }
+          },
+          {
+            "AustralianPostCode": "4069",
+            "AustralianState": "QLD",
+            "City": "Fig Tree Pocket",
+            "Country": "AU",
+            "Latitude": -27.5286302,
+            "Longitude": 152.9664001,
+            "Suburb": "Fig Tree Pocket",
+            "Type": "Standard",
+            "UniqueID": "Address_11555582",
+            "Standard": {
+              "StreetName": "Oasis",
+              "StreetNumber": "9",
+              "StreetType": "Court"
+            }
+          },
+          {
+            "AustralianPostCode": "4110",
+            "AustralianState": "QLD",
+            "City": "Pallara",
+            "Country": "AU",
+            "Latitude": -27.6174073,
+            "Longitude": 153.0116075,
+            "Suburb": "Pallara",
+            "Type": "Standard",
+            "UniqueID": "Address_12459614",
+            "Standard": {
+              "StreetName": "Ponting",
+              "StreetNumber": "95"
+            }
+          },
+          {
+            "AustralianPostCode": "2000",
+            "AustralianState": "NSW",
+            "City": "Sydney",
+            "Country": "AU",
+            "Latitude": -33.869167,
+            "Longitude": 151.2087416,
+            "Suburb": "Sydney",
+            "Type": "Standard",
+            "UniqueID": "Address_3416383",
+            "Standard": {
+              "StreetNumber": "135"
+            }
+          },
+          {
+            "AustralianPostCode": "4000",
+            "AustralianState": "QLD",
+            "City": "BRISBANE CITY",
+            "Country": "AU",
+            "Latitude": -27.4676885,
+            "Longitude": 153.0303725,
+            "Suburb": "BRISBANE CITY",
+            "Type": "Standard",
+            "UniqueID": "Address_807535",
+            "Standard": {
+              "Level": "26",
+              "StreetName": "EAGLE",
+              "StreetNumber": "111",
+              "StreetType": "Street"
+            }
+          }
+        ],
+        "BusinessChannel": {
+          "CompanyName": "TestBank",
+          "OtherIdentifier": "C9RZZZ",
+          "UniqueID": "Lender_5054",
+          "Contact": {
+            "WebAddress": "https://www.ubank.com.au/",
+            "OfficePhone": {
+              "Number": "02 9058 7402"
```

Comment:
is this a real tel number and web address, should we have a dummy tel number and web address here?